scripts/get_objf_and_derivs.py

```diff
@@ -292,3 +320,7 @@ def WriteDerivs():
     MergeCountsBackward(o)
 
 WriteDerivs()
+# Do Clean-up if needed and write whether or not the clean-up had been done
```

Comment:
.. and here too.
Or rename CleanUp to CleanUpIfNeeded().  But I prefer to test the condition before calling it.